doxygen misc warning

**Describe the bug**

cfe/fsw/cfe-core/src/inc/cfe_es.h:560: warning: expected <tr> tag but found TK_LNKWORD token instead!

cfe/fsw/cfe-core/src/inc/cfe_es.h:569: warning: The following parameters of CFE_ES_WaitForSystemState(uint32 MinSystemState, uint32 TimeOutMilliseconds) are not documented:
  parameter 'MinSystemState'

cfe/fsw/cfe-core/src/inc/cfe_es_msg.h:750: warning: Found unknown command `\B'

cfe/fsw/cfe-core/src/inc/cfe_es_msg.h:750: warning: Found unknown command `\C'

cfe/fsw/cfe-core/src/inc/cfe_fs.h:122: warning: The following parameters of CFE_FS_InitHeader(CFE_FS_Header_t *Hdr, const char *Description, uint32 SubType) are not documented:
  parameter 'Description'
  parameter 'SubType'

cfe/fsw/cfe-core/src/inc/cfe_sb.h:256: warning: argument 'OptsPtr' of command @param is not found in the argument list of CFE_SB_GetPipeOpts(CFE_SB_PipeId_t PipeId, uint8 *OptPtr)

cfe/fsw/cfe-core/src/inc/cfe_sb.h:272: warning: The following parameters of CFE_SB_GetPipeOpts(CFE_SB_PipeId_t PipeId, uint8 *OptPtr) are not documented:
  parameter 'OptPtr'

cfe/fsw/cfe-core/src/inc/cfe_sb_msg.h:560: warning: Illegal command \SB_PIPEOPTSEC as the argument of a \c command

**To Reproduce**
Steps to reproduce the behavior:
1. make usersguide

**System observed on:**
 - Hardware
 - Ubuntu 18.04
 - doxygen 1.8.13, rc-6.7.0

**Reporter Info**
Anh Van, NASA Goddard

--- a/docs/src/cfe_es.dox
+++ b/docs/src/cfe_es.dox
@@ -220,7 +220,7 @@
   cFE will also reset every application that is running at the time the command is received. 
 
   Also part of this service is the Exception and Reset (ER) Log, which has a command for 
-  \link #CFE_ES_WRITE_ERLOG_CC dumping \endlink or \link #CFE_ES_CLEAR_ERLOG_CC clearing \endlink
+  \link #CFE_ES_WRITE_ER_LOG_CC dumping \endlink or \link #CFE_ES_CLEAR_ER_LOG_CC clearing \endlink
   the log and telemetry to show the number of entries in the log.  In addition to the ER log,
    the user may find information about the most recent reset in the ES task housekeeping telemetry.
 
@@ -731,7 +731,7 @@
   allocates a section of the Critical Data Store memory for the application's use and
   assigns the Application specified name to the memory area.  The operator can find
   and learn the characteristics of these Critical Data Stores by using the
-  \link #CFE_ES_DUMP_CDS_REG_CC Dump CDS Registry Command. \endlink  This command will
+  \link #CFE_ES_DUMP_CDS_REGISTRY_CC Dump CDS Registry Command. \endlink  This command will
   dump the contents of the CDS Registry maintained by the Executive Services into a
   file that can be downlinked and examined by the operator.
   
@@ -856,7 +856,7 @@
   by mission requirements).
     
   An operator can obtain information about an Application's Memory Pool by using
-  the \link #CFE_ES_TLM_POOL_STATS_CC Telemeter Memory Pool Statistics Command.
+  the \link #CFE_ES_SEND_MEM_POOL_STATS_CC Telemeter Memory Pool Statistics Command.
   \endlink
   
   This command will cause Executive Services to extract pertinent statistics
@@ -878,7 +878,7 @@
   
   <UL>
     <LI> <B>Memory Pool Handle</B> - the handle, as provided by the operator in the
-         \link #CFE_ES_TLM_POOL_STATS_CC Telemeter Memory Pool Statistics Command. \endlink
+         \link #CFE_ES_SEND_MEM_POOL_STATS_CC Telemeter Memory Pool Statistics Command. \endlink
          This repeating of the handle in telemetry insures the operator knows which Memory
          Pool Statistics are being viewed <BR>
     <LI> <B>Pool Size</B> - The total size of the memory pool (in bytes)<BR>

--- a/docs/src/cfe_evs.dox
+++ b/docs/src/cfe_evs.dox
@@ -361,7 +361,7 @@
   however, these are only available for those events that are registered for filtering hence if you 
   have a message that is not registered for filtering and the message type (e.g. DEBUG) is disabled 
   then you won't know if the event was ever issued by an application.  These counters are available 
-  by sending a command to \link #CFE_EVS_FILE_WRITE_APP_DATA_CC write the EVS Application Data \endlink
+  by sending a command to \link #CFE_EVS_WRITE_APP_DATA_FILE_CC write the EVS Application Data \endlink
   and transferring the file to the ground. 
 
 
@@ -466,7 +466,7 @@
         <TR><TD COLSPAN=2 WIDTH="100%"> <B>(Q)
            How do I find out which events are registered for filtering?
         </B><TR><TD WIDTH="5%"> &nbsp; <TD WIDTH="95%">
-           EVS provides a command (\link #CFE_EVS_FILE_WRITE_APP_DATA_CC \EVS_WRITEAPPDATA2FILE \endlink)
+           EVS provides a command (\link #CFE_EVS_WRITE_APP_DATA_FILE_CC  \EVS_WRITEAPPDATA2FILE \endlink)
            which generates a file containing all of the applications that have registered with EVS and 
            all of the filters that are registered for each application.  Note that EVS merely generates 
            the file.  The file must be transferred to the ground in order to view it.

--- a/fsw/cfe-core/src/inc/cfe_es.h
+++ b/fsw/cfe-core/src/inc/cfe_es.h
@@ -558,6 +558,8 @@ bool CFE_ES_RunLoop(uint32 *ExitStatus);
 **                                   wait indefinitely to avoid hanging a critical
 **                                   application because a non-critical app did not start.
 **
+** \param[in]  MinSystemState        Determine the state of the App
+**
 ** \returns
 **      CFE_SUCCESS if state was successfully achieved
 **      CFE_ES_OPERATION_TIMED_OUT if the timeout was reached

--- a/fsw/cfe-core/src/inc/cfe_es_msg.h
+++ b/fsw/cfe-core/src/inc/cfe_es_msg.h
@@ -747,7 +747,7 @@
 **       the following telemetry:
 **       - \b \c \ES_CMDPC - command execution counter will 
 **         increment
-**       - \B \C \ES_PERFFLTRMASK - the current performance filter mask
+**       - \b \c \ES_PERFFLTRMASK - the current performance filter mask
 **         value(s) should reflect the commanded value
 **       - The #CFE_ES_PERF_FILTMSKCMD_EID debug event message will be 
 **         generated.

--- a/fsw/cfe-core/src/inc/cfe_es_msg.h
+++ b/fsw/cfe-core/src/inc/cfe_es_msg.h
@@ -136,7 +136,7 @@
 **  \cfecmdmnemonic \ES_RESET
 **
 **  \par Command Structure
-**       #CFE_ES_RestartCmd_t
+**       #CFE_ES_RestartCmd_Payload_t
 **
 **  \par Command Verification
 **       Successful execution of this command (as a Processor Reset)  
@@ -152,7 +152,7 @@
 **  \par Error Conditions
 **       This command may fail for the following reason(s):
 **       - The command packet length is incorrect
-**       - The \link #CFE_ES_RestartCmd_t.RestartType Restart Type \endlink was
+**       - The \link #CFE_ES_RestartCmd_Payload_t Restart Type \endlink was
 **         not a recognized value. 
 **       
 **       Evidence of failure may be found in the following telemetry:

--- a/fsw/cfe-core/src/inc/cfe_evs_msg.h
+++ b/fsw/cfe-core/src/inc/cfe_evs_msg.h
@@ -346,7 +346,7 @@
 **       - \b \c \EVS_CMDPC - command execution counter will 
 **       increment
 **       - The generation of #CFE_EVS_DISAPPENTTYPE_EID debug event message      
-**       - The clearing of the Active Flag in \link #CFE_EVS_AppDataFile_t::ActiveFlag The Active Flag in EVS App Data File \endlink
+**       - The clearing of the Event Type Active Flag in The Event Type Active Flag in EVS App Data File
 **
 **  \par Error Conditions
 **      This command may fail for the following reason(s):
@@ -386,7 +386,7 @@
 **       - \b \c \EVS_CMDPC - command execution counter will 
 **       increment
 **       - The generation of #CFE_EVS_ENAAPPEVT_EID debug event message 
-**       - The setting of the Active Flag in \link #CFE_EVS_AppDataFile_t::ActiveFlag The Active Flag in EVS App Data File \endlink
+**       - The setting of the Active Flag in The Active Flag in EVS App Data File
 **
 **  \par Error Conditions
 **      This command may fail for the following reason(s):

--- a/fsw/cfe-core/src/inc/cfe_fs.h
+++ b/fsw/cfe-core/src/inc/cfe_fs.h
@@ -113,8 +113,12 @@ int32 CFE_FS_ReadHeader(CFE_FS_Header_t *Hdr, int32 FileDes);
 **        This API will clear the specified #CFE_FS_Header_t variable and
 **        initialize the description field with the specified value
 **
-** \param[in] Hdr     Pointer to a variable of type #CFE_FS_Header_t that will be
-**                    cleared and initialized
+** \param[in] Hdr          Pointer to a variable of type #CFE_FS_Header_t that will be
+**                         cleared and initialized
+**
+** \param[in] *Description Initializes Header's Description
+**
+** \param[in]  SubType     Initializes Header's SubType
 **
 ** \sa #CFE_FS_WriteHeader
 **

--- a/fsw/cfe-core/src/inc/cfe_sb.h
+++ b/fsw/cfe-core/src/inc/cfe_sb.h
@@ -260,7 +260,7 @@ int32  CFE_SB_SetPipeOpts(CFE_SB_PipeId_t     PipeId,
 **
 ** \param[in]  PipeId       The pipe ID of the pipe to get options from.
 **
-** \param[out] OptsPtr      A bit field of options.
+** \param[out] *OptPtr      A bit field of options.
 **
 ** \returns
 ** \retcode #CFE_SUCCESS           \retdesc  \copydoc CFE_SUCCESS         \endcode

